lstchain/io/event_selection.py

```diff
@@ -0,0 +1,36 @@
+from ctapipe.core import Component
+from ctapipe.core.traits import Dict, List
+from lstchain.reco.utils import filter_events
+
+import numpy as np
+
+
+__all__ = ["EventSelector"]
+
+
+class EventSelector(Component):
+    """
+    Filter values used for event filters are taken as inputs and
+    filter_events() is used on a table of events called in with the Component.
+    """
+
+    filters = Dict(
+        help="Dict of event filter parameters",
+        default_value={
```

Comment:
These defaults look they are not cutting anything but nan values. So they should be removed and the features listed in the `finite_parameters` list.

Reply:
Ah yes! I will amend it.